deployment/manifests/demo/demo_frontendconfig.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: networking.gke.io/v1beta1
+kind: FrontendConfig
+metadata:
+  name: demo-exome-results-browsers-frontend-config
+spec:
+  redirectToHttps:
+    enabled: false
```

Comment:
You may be leaving this in here for later, but if not, you don't need this frontend config if you're not enabling the http -> https redirect, pretty sure that's off by default. If you remove this, you'll need to remove the FrontendConfig annotation from the Ingress.

Reply:
Yeah that's a good point, I had mimed the config from the production ones but then figured the redirect to http was going to cause problems with a dynamic IP being assigned, but now this does nothing. Thanks for the heads up on removing the annotation from the ingress, I was a little unsure of what changes would be required of other files if this frontend config file got removed.